cmd/presets/presets.go

```diff
@@ -193,6 +195,7 @@ networks:
 		".dockerignore": `/node_modules
 `,
 		"preset_language": "javascript",
+		"preset_create": "kooldev/node:14-nest nest new",
```

Comment:
Having the `preset_create` standard like this, making it mandatory to run only `kool docker` commands, I guess is an unnecessary constraint... why can't we simply leave it for the user the whole command? For example all these could simple be prefixed with `kool docker <image> <commands...>`, giving it more flexibility IMO.